daft/expressions/expressions.py

```diff
@@ -1179,6 +1179,19 @@ def find(self, substr: str | Expression) -> Expression:
         substr_expr = Expression._to_expression(substr)
         return Expression._from_pyexpr(self._expr.utf8_find(substr_expr._expr))
 
+    def rpad(self, length: int | Expression, pad: str | Expression) -> Expression:
+        """Right-pads each string by truncating or padding with the character
+
+        Example:
+            >>> col("x").str.rpad(5, "0")
+
+        Returns:
+            Expression: a String expression which is `self` truncated or right-padded with the pad character
+        """
```

Comment:
You should add in the docstring something explicitly saying that the `pad` parameter must be a single character or an expression consisting of single characters

Reply:
added